src/util/ThreadMemberFlags.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const BitField = require('./BitField');
+
+/**
+ * Data structure that makes it easy to interact with a {@link ThreadMember#flags} bitfield.
+ * @extends {BitField}
+ */
+class ThreadMemberFlags extends BitField {}
+
+/**
+ * @name ThreadMemberFlags
+ * @kind constructor
+ * @memberof ThreadMemberFlags
+ * @param {BitFieldResolvable} [bits=0] Bit(s) to read from
+ */
+
+/**
+ * Bitfield of the packed bits
+ * @type {number}
+ * @name ThreadMemberFlags#bitfield
+ */
+
+/**
+ * Numeric thread member flags. There are currently no bitflags relevant to bots for this.
+ * @type {Object}
```

Comment:
```suggestion
 * @type {Object<string, number>}
```

Reply:
Is this actually valid JSDoc?

Reply:
It at least seems to work as intended:
![image](https://user-images.githubusercontent.com/24881032/122680894-de30dd80-d1f1-11eb-8aff-61763a369d18.png)

Reply:
> Is this actually valid JSDoc?

Yes, although the correct type would be `Record`. You can even do `import('').Type` and it'd work (but our site doesn't support that)

Reply:
https://jsdoc.app/tags-type.html